README.md

```diff
@@ -1,7 +1,8 @@
 # Snakemake workflow: cyrcular-calling
 
 [![Snakemake](https://img.shields.io/badge/snakemake-≥6.15.0-brightgreen.svg)](https://snakemake.github.io)
-
+[![Tests](https://github.com/snakemake-workflows/cyrcular-calling/actions/workflows/main.yaml/badge.svg)](https://github.com/snakemake-workflows/cyrcular-calling/actions/workflows/main.yaml)
+[![Conventional Commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-%23FE5196?logo=conventionalcommits&logoColor=white)](https://conventionalcommits.org)
```

Comment:
If zenodo is already enabled, we should add the respective badge as well

Reply:
We only get it once we have an initial release. So we'll have to add it in another PR.